supply_water.md

```diff
@@ -0,0 +1,112 @@
+
+### Water subsidies and water rights
+
+The water distribution sector for each demand category includes the cost of water transport, distribution and/or treatment and other adjustments such as subsidies, in addition to the resource cost of water. Thus, water prices for each demand category is differentiated. This framework provides a covenient location for including water subsidies or reduced water costs from allocation rights for irrigated agriculture. In GCAM, the price of water for agriculture use has been reduced by a factor of 1/100th. This factor is based on an OECD report on water pricing ([OECD 2009](#oecd2009)) which indicates that industrial and household water users pay more than 100 times as much as agriculture users. In some regions, the marginal cost of irrigated water use is zero. Crop growth requires substantial water inputs and the profitability of agricultural relies on the low cost of water provisions. Without this subsidy, historical calibration of agriculture production in GCAM is not possible for historically water scarce regions with high cost of water. Water subsidies for irrigated agriculture have not been differentiated by region and basin, and improved understanding of regional differences in water subsidies and their impact on agriculture production remain for future work.
+
+## Equations 
+The equations that determine water supply are described here.
+
+#### Variable #1
+
+{Insert equations used to calculate variable #1}
+
+See `method name` in [code_file.cpp](link to code on GitHub).
+
+## Policy options 
+This section summarizes some of the energy-based policy options available in GCAM. 
+
+### Policy type #1
+
+{<}Insert paragraph describing a type of policy for this sector. Include links to xml input files on GitHub and/or the policies.html or policies_examples.html pages}
+
+## Insights and intuition
+
+### Paper/Topic #1
+
+{One paragraph summary of a key insight from one or more papers}
+
+## IAMC Reference Card
+
+Co-linkages
+- [X] Water availability
+
+
+<a name="table_footnote">1</a>: Note that this table differs from the one provided on the [Supply Inputs Page](inputs_supply.html#description) in that it lists all inputs to the land model, including information passed from other modules. Additionally, the units listed are the units GCAM requires, rather than the units the raw input data uses.
+
+## References
+
+<a name="cui2018">[Cui et al. 2018]</a> Cui, Ryna Yiyun, Katherine Calvin, Leon Clarke, Mohamad Hejazi, Son Kim, Page Kyle, Pralit Patel, Sean Turner and Marshall Wise (2018). *Regional responses to future, demand-driven water scarcity*. Environmental Research Letters, 13, 9. [Link](https://doi.org/10.1088/1748-9326/aad8f7)
+
```

Comment:
add:

<a name="eurek2017">[Eurek et al. 2017]</a> Eurek, K., P. Sullivan, M. Gleason, D. Hettinger, D. Heimiller, A. Lopez (2017). *An improved global wind resource estimate for integrated assessment models*. Energy Economics, 64, [Link](https://doi.org/10.1016/j.eneco.2016.11.015)